grpc-interface-support/src/cpp.py

```diff
@@ -17,20 +17,356 @@
 import shutil
 import subprocess
 import tempfile
+from io import TextIOWrapper
 from pathlib import Path
+from typing import Callable, Dict, List, Optional
 
-from generator import GrpcInterfaceGenerator
+import shared_utils
```

Comment:
please choose only one: import all or define the needed types/methods (line 27)

Reply:
Good catch. Done.